models/src/constants.rs

```diff
@@ -1,3 +1,5 @@
+// TODO - need to refactor constants
```

Comment:
Is this TODO intentional?

Reply:
Probably not - I need to go back again after the cron PR merges to do another round of refactoring / rebase to get those into the helm chart